images/manageiq-base/container-assets/create_local_yum_repo.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+yum -y install createrepo_c
+createrepo /tmp/rpms
+yum -y remove createrepo_c
+
+cat > /etc/yum.repos.d/local_rpm.repo << EOF
+[local-rpm]
+baseurl=file:///tmp/rpms/$basearch
+name=Local yum repo
+enabled=1
+gpgcheck=0
+EOF
+
+dnf config-manager --setopt=copr:copr.fedorainfracloud.org:simaishi:test.exclude=manageiq-* --save
```

Comment:
What will this look like when it's not your personal repo?

Reply:
`--setopt=copr:copr.fedorainfracloud.org:manageiq:ManageIQ-Master.exclude=manageiq-*`

But we will have to add support for customization, for both the repo name and the RPM name.

Reply:
Probably can extract those from the options.yml for the build.